src/App.test.js

```diff
@@ -1,8 +1,65 @@
 import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import App from './App';
 
-test('renders learn react link', () => {
+test('renders home title', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+  const title = screen.getByRole('heading', /grocery list/i);
+  expect(title).toBeInTheDocument();
+});
+
+test('renders list of items', () => {
+  render(<App />);
+
+  const items = screen.getAllByRole('listitem');
+  expect(items).toHaveLength(3);
+});
+
+test('add button will add new item to list', () => {
+  render(<App />);
+
+  const input = screen.getByLabelText(/form input/i);
+  userEvent.type(input, 'tater tots');
+
+  const button = screen.getByRole('button', { name: /add \+/i });
+  userEvent.click(button);
+
+  expect(screen.getByText(/tater tots/i)).toBeInTheDocument();
+});
+
+test('delete button will delete item from list', () => {
+  render(<App />);
+
+  const oatMilk = screen.getByText(/oat milk/i);
+  const deleteButton = screen.getByTestId('edit-0');
```

Comment:
Here's the cheatsheet and Priority lists linked today in lecture. Sounds like an aria labels might be a suitable replacement for getByTestId on these buttons. 
https://testing-library.com/docs/react-testing-library/cheatsheet/
https://testing-library.com/docs/queries/about/#priority